cmd/lotus-shed/migrations.go

```diff
@@ -240,7 +240,7 @@ func checkMigrationInvariants(ctx context.Context, v8StateRootCid cid.Cid, v9Sta
 		return err
 	}
 
-	v9actorTree, err := builtin.LoadTree(actorStore, v9stateRoot.Actors)
+	v9actorTree, _ := builtin.LoadTree(actorStore, v9stateRoot.Actors)
```

Comment:
We should definitely check the error here.

Reply:
And do what? Panic?

Reply:
Just return it

`	if err != nil {
		return err
	}`